app/views/simulator/edit.html.erb

```diff
@@ -4,6 +4,8 @@
 <!-- <link rel="stylesheet" href="https://code.jquery.com/ui/1.12.1/themes/base/jquery-ui.css"> -->
 <!-- <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css" integrity="sha384&#45;BVYiiSIFeK1dGmJRAkycuHAHRg32OmUcww7on3RYdg4Va+PmSTsz/K68vbdEjh4u" crossorigin="anonymous"> -->
 <link rel="stylesheet" href="https://code.jquery.com/ui/1.12.1/themes/base/jquery-ui.css">
+<link href="https://fonts.googleapis.com/css?family=Fira+Sans|Roboto+Mono&display=swap" rel="stylesheet">
+<link rel="stylesheet" type="text/css" href="/css/sidebar.css">
```

Comment:
try not to create new stylesheets, add styles to dedicated stylesheets.

Reply:
yea , okay fine